nix/shell.nix

```diff
@@ -0,0 +1,25 @@
+{ pkgs ? (import <nixpkgs>) { } }:
+let
+  # `volatile` is not packaged in nix.
+  volatile = pkgs.python38Packages.buildPythonPackage rec {
+    pname = "volatile";
+    version = "2.1.0";
+    src = pkgs.python38Packages.fetchPypi {
+      inherit pname version;
+      sha256 = "1lri7a6pmlx9ghbrsgd702c3n862glwy0p8idh0lwdg313anmqwv";
+    };
+    doCheck = false;
+  };
+  python = pkgs.python3.withPackages
+    (python-packages: with python-packages; [ click kubernetes volatile ]);
+in pkgs.mkShell {
+
+  buildInputs = with pkgs; [ skopeo kubectl python black ];
```

Comment:
Why do we need `black`?  It's just a formatter, right?

Reply:
The nix scripts in general need a bit of cleanup, but here it is almost justifiable. My general idea is that `shell.nix` == convenient development environment, while `default.nix` (or in this case, `node.nix`) is the cleaner packaging.

I have "misused" the `shell.nix` to include tooling that is used to develop `casper-tool` as well. Short to midterm, I would like to remove some of these scripts altogether and have a unified "development environment for the whole project" `shell.nix` on the root, like I have on my local machine.